app/services/AuthService.ts

```diff
@@ -13,5 +17,27 @@ export const AuthService = {
 			return true;
 		}
 		return false;
+	},
+
+    async validateLogin(uid: string, password) {
+
+		
+	    var data
+		await axios.post('hslinux:38383/api/v1/auth', {
+		uid: uid,
+		password: password
+		  })
+		  .then(function (response) {
+		data = console.log(response.data);
```

Comment:
You need to return the data that is being responded back from the axios. In this line you are printing the response.data to the screen and not storing it in data(I think). Regardless you do not need to console.log the response data